operators/Makefile

```diff
@@ -17,7 +17,7 @@ GKE_CLUSTER_VERSION ?= 1.12
 
 REPOSITORY 	?= eck
 NAME       	?= eck-operator
-VERSION    	?= 0.9.0-SNAPSHOT
+VERSION    	?= $(shell cat VERSION)
```

Comment:
Environment variable does not work for CI?

Reply:
Works, but in this case we will need to support and sync version in several files (current Makefile and one or more Jenkins pipelines) which makes it's hard and error prone. It's better when it's in one place